draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -340,17 +377,22 @@ Vary: accept-encoding, available-dictionary
 
 ## Content Encoding
 
-IANA is asked to enter the following into the "HTTP Content Coding Registry" registry ({{HTTP}}):
+IANA is asked to enter the following into the "HTTP Content Coding Registry"
+registry ({{HTTP}}):
 
 - Name: br-d
-- Description: A stream of bytes compressed using the Brotli protocol with an external dictionary of not more than 16 MB.
+- Description: A stream of bytes compressed using the Brotli protocol with an
+external dictionary and a compression window of not more than 16 MB.
```

Comment:
Now that the window size is more complex, I would drop this additional text.  Instead, reference the definition of the window size above.  (I would structure this differently and have sections for each algorithm, that includes a discussion of window size, rather than sections for the window size.)

Reply:
It seems nice to keep the requirement itself in the IANA registry, and have the longer discussion and reasoning up above.

Reply:
I changed it to be a named format instead of trying to include all of the requirements and settings directly in the IANA registry.  They're now referenced as `Dictionary-Compressed Brotli` and `Dictionary-Compressed Zstandard`